pkg/controller/migmigration/restore.go

```diff
@@ -280,17 +310,27 @@ func (t *Task) hasRestoreCompleted(restore *velero.Restore) (bool, []string) {
 			restore.Namespace,
 			restore.Name)
 		reasons = append(reasons, message)
+		itemsRestored, totalItems := getRestoreStats(restore)
+		message = fmt.Sprintf(
+			"%s %d out of estimated total of %d objects restored%s",
+			message,
+			itemsRestored,
+			totalItems,
+			getRestoreDuration(restore))
 		progress = append(progress, message)
 		progress = append(
 			progress,
 			getPodVolumeRestoresProgress(pvrs)...)
 	case velero.RestorePhasePartiallyFailed:
 		completed = true
+		itemsRestored, totalItems := getRestoreStats(restore)
 		message := fmt.Sprintf(
-			fmt.Sprintf(
-				"Restore %s/%s: partially failed.",
-				restore.Namespace,
-				restore.Name))
+			"Restore %s/%s: partially failed. %d out of estimated total of %d objects restored%s",
```

Comment:
If you make the whitespace changes suggested above, the space after "restored" is needed here too.